examples/intermediate/cross_core.py

```
@@ -0,0 +1,268 @@
+    parser.add_argument("-p", "--platform", type=str, default="a2a3",
+                        choices=["a2a3", "a2a3sim", "a5", "a5sim"])
+    parser.add_argument("-d", "--device", type=int, default=0)
+    parser.add_argument("--chunk", type=int, default=4, choices=[1, 2, 4, 8, 16],
+                        help="Chunk size for parallel loop (smaller = more parallel tasks)")
+    parser.add_argument("--no-fusion", action="store_true",
+                        help="Use unfused version (separate pl.at blocks)")
+    parser.add_argument("--runtime-profiling", action="store_true", default=False)
+    args = parser.parse_args()
+
+    print(f"\n{'='*60}")
+    print(f"Cross Core — Stage 0 & 1 Fusion Test")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)